src/MicroOrm.Dapper.Repositories/MicroOrm.Dapper.Repositories.csproj

```diff
@@ -30,12 +32,16 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.*" PrivateAssets="All" Condition="'$(OS)' != 'Windows_NT'" />
+    <PackageReference Include="Nullable" Version="1.*" PrivateAssets="All" />
```

Comment:
Why do we need this?

Think enough just:
```c#
<Nullable>enable</Nullable>
```

Reply:
Some type not exits, like `[MemberNotNullWhen]`.

Reply:
`Microsoft.NETFramework.ReferenceAssemblies' fix build project on non-windows.